nomenclature/definitions/variable/economy/economy.yaml

```diff
@@ -4,6 +4,66 @@ Consumption:
    description: Total consumption of all goods, by all consumers in a region
    unit: billion US$2010/yr
 
+Consumption|Households|<Product>|Imported:
+   description: Consumption of imported <this product> by households
+   unit: million euro/yr
+
+Consumption|Government|<Product>|Domestic:
+   description: Consumption of domestic <this product> by households
+   unit: million euro/yr
+
+Consumption|Government|<Product>|Imported:
+   description: Consumption of imported <this product> by government
+   unit: million euro/yr
+
+Consumption|Government|<Product>|Domestic:
+   description: Consumption of domestic <this product> by government
+   unit: million euro/yr
+
+Consumption|GrossCapitalFormation|<Product>|Imported:
+   description: Consumption of imported <this product> by gross capital formation
+   unit: million euro/yr
+
+Consumption|GrossCapitalFormation|<Product>|Domestic:
+   description: Consumption of domestic <this product> by gross capital formation
+   unit: million euro/yr
+
+Consumption|Industry|<Product>|Imported:
+   description: Consumption of imported <this product> by industries
+   unit: million euro/yr
+
+Consumption|Industry|<Product>|Domestic:
+   description: Consumption of domestic <this product> by industries
+   unit: million euro/yr
+
+IndustryOutput|<Industry>:
+   description: Total output on <this industry> level
+   unit: million euro/yr
+
+ProductOutput|<Product>:
```

Comment:
Please rename to `Output|Product|<Product>`, and is there a need for aggregate product output as well?

Reply:
@hettieboonman, any comment?

Reply:
Sorry, I missed this question. There is indeed no need for an aggregate product output or aggregated industry output.

Reply:
thank you, resolving this comment to mark it as closed